scripts/code-coverage.sh

```diff
@@ -17,12 +17,16 @@ if [[ ! $2 = "iOS" && ! $2 = "Android" ]]; then
     exit 1
 fi
 
+circle_sha=""
+if [[ $CIRCLE_SHA1 ]]; then
+    circle_sha="$CIRCLE_SHA1"
+fi
 # Create a formatted JSON file that contains the current coverage. 
 
 current_date=$(TZ=UTC date +"%Y-%m-%d")
```

Comment:
Let's include the time in here too, as we have multiple merges per day: `date -u +%FT%TZ` seems to do the trick.